src/main/java/biblioteket/roborally/programcards/ICard.java

```diff
@@ -25,4 +25,9 @@
      * @param robot which robot that should do the action.
      */
     void doCardAction(IRobot robot, IBoard board);
+
+    /**
+     * @return a copy of this card
+     */
+    ICard clone();
```

Comment:
Kanskje heller kalle metoden `copy()` siden du lager en kopi og ikke kloner (to litt forskjellige ting i programmering).